src/go/k8s/config/rbac/role.yaml

```diff
@@ -36,6 +46,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
```

Comment:
Is the `watch` missing?

Reply:
I don't think we need to watch secrets at this point, do we?